### DISCUSSION TITLE
Lesson 7: Source not found, File import callback not supported

### DISCUSSION THREAD

**Discussion by limrik** [Cyfrin/foundry-full-course-cu#1745] [Q&A]
Hi, I am facing issues regarding file importing, have included remappings as well but that does not seem to work. 

<img width="752" alt="Screenshot 2024-05-23 at 11 01 35 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/64995992/270d3418-86c3-4b21-a6d8-11ad535f4cc3">

**Comment by EngrPips** [comment]
Hello @limrik, Have you installed the `forge-std` library already?

**Comment by urosognjenovic** [comment]
You seem to have created DeployFundMe.s.sol in ```lib/forge-std/scripts/``` instead of the root ```scripts/``` folder. That's what causes the error on the second import.

As for the first import, could you please run ```forge remappings``` in the command line and share what you get with us? There should be ```forge-std/=lib/forge-std/src/``` included in the results for the first import to work.

**Comment by EngrPips** [comment]
Hello, @limrik. Did you resolve this issue already?

**Comment by footprns** [comment]
Hi @limrik , please make sure that you run `forge init` at the beginning of your project. The `forge-std` will loaded automatically

**Comment by limrik** [comment]
Hi @thebraveman17 and @footprns, thank you for the input, but in the end I decided to redo lesson 7 as I was not very far in anyways.

**Reply by EngrPips** [reply]
Excellent, I am Glad you, one way or the other, unblocked yourself.
